README.md

```diff
@@ -15,7 +15,7 @@ Installation
 
 `composer require dividebv/phpdivideiq`
 
-Example Usage
+Example Usage Stock GET
```

Comment:
Left it as it was, fixed this in commit: 
https://github.com/DivideBV/PHPDivideIQ/pull/1/commits/6d4e7e5a4357760aea9a78bef68801403b1e5440